docs/getting_started/tutorial.md

```diff
@@ -84,10 +92,8 @@ $ cd mysite
 $ pip install -r requirements.txt
 ```
 
-This ensures that you have the relevant versions of
-Wagtail,
-Django,
-and any other dependencies for the project you have just created.
+the `requirements.txt` file contains all the dependencies needed
+in order to run the project.
```

Comment:
@Dev-Yusuf happy for us to consider changes to this but I felt like the existing content was useful. Perhaps we could add your content in addition, rather than instead?

```suggestion
This ensures that you have the relevant versions of Wagtail, Django, and any other dependencies for the project you have just created.
The `requirements.txt` file contains all the dependencies needed in order to run the project.
```